api/src/devcomp/infrastructure/datasources/learning-content/modules/ia-deepfakes.json

```
@@ -0,0 +1,537 @@
+{
+  "id": "8aa17e6d-3470-479d-838d-ff6923de6686",
+  "slug": "ia-deepfakes",
```

Pour s'éviter des problèmes : 

```suggestion
  "slug": "tmp-ia-deepfakes",
```